test/test_alexa_integration_express.js

```diff
@@ -102,6 +113,24 @@ describe("Alexa", function() {
             expect(response.text).to.startWith('{"name":"testApp"')
           });
       });
+
+      it("returns debug schema", function() {
+        return request(testServer)
+          .get('/testApp?schema')
+          .expect(200).then(function(response) {
+            expect(response.headers['content-type']).to.equal('text/plain; charset=utf-8');
+            expect(response.text).to.include('"intent": "myIntent"')
```

Comment:
It would be safer to compare response.text to the actual result. If the result is really big, you can load it from a file and compare from that. See [alexa-app-server/index.js#L85](https://github.com/alexa-js/alexa-app-server/blob/master/test/test-examples-server.js#L85) for an example.